projects/rero/ng-core/src/lib/record/search/record-search-page.component.ts

```diff
@@ -215,12 +216,20 @@ export class RecordSearchComponent implements OnInit, OnDestroy {
     this._router.navigate([this.getCurrentUrl(parameters.currentType)], { queryParams, relativeTo: this._route });
   }
 
+  /**
+   * Result of search with type and records (Elasticsearch)
+   * @param event, EsResult content type and records
+   */
+  recordsResult(event: EsResult) {
+    // Implement this function on your extended class.
+  }
+
```

Comment:
I think you should not add this method in ng-core, only in extended class.

Reply:
done